SWLOR.Game.Server/Service/CraftService/RecipeType.cs

```diff
@@ -607,6 +607,53 @@ public enum RecipeType
         FabricationTerminal = 1203,
         Refinery = 1204,
         SmitheryBench = 1205,
+        RugClassicLightBrown = 1206,
+        PileofCushionsWhite = 1207,
+        Oven = 1208,
+        CoffeeMaker = 1208,
+        MicrowaveBlack = 1209,
+        BarWooden = 1210,
+        CouchLeatherBlue = 1211,
+		BenchLeatherCushionBrown = 1212,
+		TablePlasticLarge = 1213,
+        TableRoundOak = 1214,
+        MetalWallWideLight = 1215,
+        StoreCounterStained = 1216,
+        ChairLargeGreyRed = 1217,
+        BunkbedMetalGrey = 1218,
+        ShowerWhite = 1219,
+        ShowerFloorBasin = 1220,
+        CouchLeatherPanelsGrey = 1221,
+        ShelvesWarehouseFull = 1222,
+        BookshelfJedi = 1223,
+        MetalWallSinglePipes = 1224,
+        WoodenWallPlanksSmall = 1225,
+        TelevisionBigScreen = 1226,
+        TelevisionOldModel = 1227,
+        FoyerChandelier = 1228,
+        MetalWallDoorLight = 1229,
+        MetalWallWidePanels = 1230,
+        MetalWallDoorPanels = 1231,
+        MetalWallSingleLight = 1232,
+        MetalWallSingleRibbed = 1233,
+        MetalWallWideRibbed = 1234,
+        MetalWallDoorRibbed = 1235,
+        MetalWallWidePipes = 1236,
+        MetalWallDoorPipes = 1237,
+        CouchCushionGreyRed = 1238,
+        BenchElegantGrey = 1239,
+        ChaiseLoungeOrange = 1240,
+        CouchBlanketCoverRed = 1241,
+        ChairPlinth = 1242,
+        ChairDiningGrey = 1243,
+        ChaiseLoungeRed = 1244,
+        ChairDiningOrange = 1245,
+        TablePolygonDesign = 1246,
+        TableRoundGlass = 1247,
+        TableDarkGlass = 1248,
+
```

Comment:
nitpick: You have a bunch of extra empty lines here that can be deleted.

Reply:
note to self: corrected. followed formatting layout to put // Structures: 1496-1516 above following segment for structure category